components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Twin/src/Control/Services/AddressSpaceServices.cs

```diff
@@ -868,13 +868,18 @@ public sealed class AddressSpaceServices : INodeServices<EndpointModel>,
                             OperationResultEx.Validate("FetchChildren_" + nodeId,
                                 diagnostics, response?.Results?.Select(r => r.StatusCode),
                                 response?.DiagnosticInfos, true);
-                            if (response.Results.Count == 1) {
+                            if (response.Results.Count > 0) {
                                 children = response.Results[0].References.Count != 0;
+                                if (response.Results[0].ContinuationPoint != null) {
+                                    await session.BrowseNextAsync(header, true,
+                                        new ByteStringCollection {
+                                            response.Results[0].ContinuationPoint
+                                        });
```

Comment:
Why do we force here a browse next? just to release the provided continuation point? in that case maybe we should do that also when no results are returned (count == 0) but a valid continuation point - would be super weird, but you never know ...

Reply:
Thought the same and had that first - but found that really super weird.   But yes, this is interesting to trace.